### ISSUE TITLE
 Single projection for all feature maps?

### ISSUE THREAD

**Issue by zwx8981** [luyao777/HBP-Pytorch#3]
I think you should use three individual linear transforms to project Conv5_1, Conv5_2 and Conv5_3 respectively according to the original paper.

**Comment by luyao777** [comment]
Hi, thanks for your issue.
I have tried this method before, but it didn't work :(

**Comment by zwx8981** [comment]
It seems that they didn't use batch normalization nor relu in the bilinear project module.

**Comment by luyao777** [comment]
After the New Year, I tried several methods you proposed, but the results were worse than the current result.
The last layer training is about 78.96%

**Comment by twmht** [comment]
@luyao777 

你有跑過作者的 caffe 版本，可以達到作者的準確度嗎？

**Comment by luyao777** [comment]
我跑的結果接近作者的准確率

**Comment by zwx8981** [comment]
我觉得你可以尝试一下改变学习率，感觉pytorch的学习率设置跟caffe不太一样，我个人的经验是pytorch需要设置更大的学习率。
